Could not compile probe-run v0.1.6 on macOS 10.15.7

So far I have installed probe-run without problems on macOS (x86_64) with:
```
cargo install --git https://github.com/knurling-rs/probe-run --branch main --features defmt
```

Now I tried to install release version with:
```
cargo install probe-run
```

And got following error:
```
  = note: Undefined symbols for architecture x86_64:
            "_NSAppKitVersionNumber", referenced from:
                _hid_init in libhidapi-6c82a0d52f552184.rlib(hid.o)
                _hid_enumerate in libhidapi-6c82a0d52f552184.rlib(hid.o)
                _hid_open_path in libhidapi-6c82a0d52f552184.rlib(hid.o)
          ld: symbol(s) not found for architecture x86_64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)


error: aborting due to previous error

error: failed to compile `probe-run v0.1.6 (https://github.com/knurling-rs/probe-run?branch=main#013c0bd3)`, intermediate artifacts can be found at `/var/folders/5b/jlxfj70x1fb_n2l6qh21xxfh0000gn/T/cargo-installcEziMp`

Caused by:
  could not compile `probe-run`.

To learn more, run the command again with --verbose.
```

[Full log](https://github.com/knurling-rs/probe-run/files/5601704/probe.log)

Rust version:
```
rustc 1.46.0 (04488afe3 2020-08-24)
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -30,6 +30,9 @@ rustc-demangle = "0.1.16"
 signal-hook = "0.1.16"
 structopt = "0.3.15"
 
+# pin version to avoid issue #113 on macOS
+hidapi = "=1.2.3"
+
 [features]
 # this feature does nothing and it's kept for backwards compatibility; it will be removed in the next semver bump
 defmt = []

